en/docs/administer/configuring-a-jdbc-user-store.md

```diff
@@ -1,163 +1,174 @@
-    authentication information) as well as the User Store Manager (for
-    defining users and roles). In the case of the WSO2 Identity Server, the
-    default user store is an LDAP (Apache DS) that is shipped with the
-    product.
-
-When you configure a JDBC user store as the primary user store, you can
-either use the default configuration or you can change it in the
-following ways:
-
--   You can set up two separate databases for the Authorization Manager
-    and the User Store Manager.
--   It is not recommended to use the default H2 database in production.
-    Therefore, you can replace this. For instructions on replacing this
-    by setting up a new RDBMS and configuring it for your system, see
-    [Setting Up the Physical
-    Database](../../administer/working-with-databases).
-
-Therefore, before you begin, ensure that the RDBMS that you want to use
-as the JDBC user store is correctly set up for your system. Then, follow
-the steps given below to configure a JDBC user store as the primary user
-store in your product.
-
-### Step 1: Configuring the JDBC user store manager
-
-
-To configure a JDBC user store as the primary user store, you must
+    The primary user store that is configured by default in WSO2 Identity Server
+    is an LDAP user store, This database is used by the Authorization Manager (for user authentication
+    information) as well as the User Store Manager (for defining users and
+    roles).
+
+We can specify the type of user store that we are using in the `         deployment.toml       ` file. 
+If we specified it as a JDBC user store by default,  For detailed guide on configuring 
+RDBMS database as a datasource see  [Working With Databases](../../administer/working-with-databases).
+
+### Step 1: Changing the default User store.
+
+To configure a JDBC user store as the primary user store, you need to 
 change the `         [user_store]        ` section in the
 `         <IS_HOME>/repository/conf/deployment.toml        ` file.
 
-1.  Uncomment the following section:
+1. Replace the default `user_store` configuration as follows.
 
-    ``` xml
+    ``` toml
     [user_store]
-    class = "org.wso2.carbon.user.core.jdbc.JDBCUserStoreManager"
+    type = "database"
     ```
+   By default it will use the  `database.shared_db` 
+   configurations in the `         deployment.toml       `  file. As the datasource configuration.
+   
+   If you have a requirement of changing the `database.shared_db` configuration see 
+   [Working With Databases](../../administer/working-with-databases).
+   
+   `database.shared_db` is used to store the both registry and user management related data. If you 
+   are having a requirement of using a datasource specified for the user management please follow . Otherwise you do 
+   step 2.
+    
+2. Specify the datasource that you want to use.
+    
+    1. Create a database on a [any supported RDBMS database](../../administer/working-with-databases). 
```

Comment:
Create a new database. This can be [any of the supported RDBMS databases](../../administer/working-with-databases).